### PR TITLE
Add images scales vocabulary

### PR DESCRIPTION
**Pull request by bsuttor**
related to issue https://github.com/plone/plone.app.vocabularies/issues/11

and usefull for adding settings for leadimage behavior

### CODE REVIEW

File: plone/app/vocabularies/images.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from zope.interface import provider
+
+
+@provider(IVocabularyFactory)
+def ScalesVocabulary(context):
+    """Obtains available scales from plone.app.imaging
+    """
+    terms = []
+    for scale, (width, height) in getAllowedSizes().iteritems():
+        terms.append(SimpleTerm(scale, scale, "{:s} {:d}x{:d}".format (
```
Comment (jensens) [inline plone/app/vocabularies/images.py]: - please use always numbers for the positions, like {0:s} {1:s}
- here some dynamic i18n message ids would be very useful, like ``_('imagescale_{0:s}.format(scale)', default='{0:s} ${width}x${height}'.format(scale), mapping={'width': str(width), 'height': str(height)})``` (untested)

**Comment by jensens** [comment]
- this should go into master first
- documentation in README.rst and in plone documentation (if there is any about vocabularies) is missing.
- now this package depends on plone.app.imaging, this should be reflected in setup.py
- needs check if dependency on p.a.imaging introduces some circular dependency (if so this vocabulary would better live at p.a.imaging)

**Comment by bsuttor** [comment]
I don't knpw how to check circular dependency, I installed a fresh Plone 5 with my changes (plone.app.imaging added into setup.py), and I didn't see circular dependency in my './bin/instance fg' background.

**Comment by jensens** [comment]
I just checked, there are no circular deps.

**Comment by thet** [comment]
before this is getting merged, the commits should be squashed.
/cc @bsuttor

**Comment by bsuttor** [comment]
@thet 
Is this what's you want ? It's my first time I squash commit : https://github.com/plone/plone.app.vocabularies/pull/29/commits/ba5f6e7c88f8de164c4015a6543e9d846fa774fc

**Comment by jensens** [comment]
@bsuttor github allows squash merges now through the UI too, but your manual squash looks fine too. https://github.com/blog/2141-squash-your-commits

**Pull request merged**